composer.json

```diff
@@ -95,6 +95,12 @@
         "squizlabs/php_codesniffer": "~3.6.0",
         "symfony/finder": "^5.2"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/eduard13/magento2-functional-testing-framework"
```

Comment:
Is this legal?

Reply:
Personally, don't see any problem of using a such approach for testing purposes, since the MFTF changes weren't merged yet.

Reply:
It's ok if PR marked as WIP, but little bit confuse when I see `Progress: review`